sysutils/mcfly/Portfile

```diff
@@ -0,0 +1,354 @@
+PortSystem          1.0
+PortGroup           cargo 1.0
+PortGroup           github 1.0
+
+categories          sysutils
+
+github.setup        cantino mcfly 0.2.5 v
+
+platforms           darwin
+supported_archs     x86_64
+
+license             MIT
+maintainers         nomaintainer
+description         Fly through your shell history.
+long_description    Fly through your shell history. Great Scott!
```

Comment:
The description isn't very useful. The long description tells me nothing else. Decent descriptions are necessary because otherwise people looking at a package can't figure out why they would want to use it.